mk/container.mk

```diff
@@ -0,0 +1,19 @@
+PHONY: build-run clean logs stop help
```

Comment:
I see the following error when running `make container clean` :

```
❯ make container clean
build-run                                 Build the Docker Image & then run the containers
clean                                     Remove all the containers along with volumes
help                                      Prints this message
logs                                      View logs of the all the containers
stop                                      Stop all the containers
make: *** No rule to make target `clean'.  Stop.
```

Reply:
it would be like 
`make container target=clean`
https://github.com/astronomer/astro-sdk/pull/597/files#diff-4fd9f56e707d4bf8c35dab69efacc13669aa6cbf743ee73409233a9bc6fc6630R170

Reply:
```
❯ make container target=clean
container.mk:6: *** missing separator.  Stop.
make: *** [container] Error 2
```

Reply:
I now get the above error @pankajastro ^^

Reply:
weird, let me take the latest code and check it.

<img width="664" alt="Screenshot 2022-08-09 at 7 52 07 PM" src="https://user-images.githubusercontent.com/98807258/183673830-98b0c19a-c6e0-4dc9-8325-25c0ca61fe8d.png">